x/auth/legacy/v043/store.go

```diff
@@ -0,0 +1,178 @@
+package v043
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/exported"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/gogo/protobuf/grpc"
+	"github.com/gogo/protobuf/proto"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+const (
+	delegatorDelegationPath = "/cosmos.staking.v1beta1.Query/DelegatorDelegations"
+	balancesPath            = "/cosmos.bank.v1beta1.Query/AllBalances"
+)
+
+func migrateVestingAccounts(ctx sdk.Context, accounts []types.AccountI, queryServer grpc.Server) ([]types.AccountI, error) {
+	for i := 0; i < len(accounts); i++ {
+		asVesting := vesting(accounts[i])
+		if asVesting == nil {
+			continue
+		}
+
+		account := accounts[i]
```

Comment:
for acc := range accounts {
		asVesting, ok := acc.(exported.VestingAccount)
		if !ok {
			continue
		}

Reply:
and we can remove `func vesting` which saves as 11 lines of code in total

Reply:
Good call, will remove it.